aima-core/src/main/java/aima/core/agent/DTAgent.java

```diff
@@ -0,0 +1,62 @@
+package aima.core.agent;
+
+
+import aima.core.agent.Action;
+import aima.core.agent.Percept;
+import aima.core.agent.impl.AbstractAgent;
+import aima.core.probability.BeliefState;
+
+import java.util.List;
+
+/**
+ * Artificial Intelligence A Modern Approach (3rd Edition): Figure 13.1, page 484.<br>
+ * 
+ * Figure 13.1  A decision-theoretic agent that selects rational actions.
+ * 
+ * @author Ritwik Sharma
+ */

```

Comment:
Always add the name of all the authors from whom you have copied the code. In this case, the code is entirely similar to the AIMA4e branch. So, you should give the credit to the person who wrote it (even if you have copied just a single line).